pkg/instancetype/instancetype.go

```diff
@@ -767,15 +812,15 @@ func (m *InstancetypeMethods) inferDefaultsFromVolumes(vm *virtv1.VirtualMachine
 		if volume.DataVolume != nil {
 			return m.inferDefaultsFromDataVolume(vm, volume.DataVolume.Name, defaultNameLabel, defaultKindLabel)
 		}
-		return "", "", fmt.Errorf("unable to infer defaults from volume %s as type is not supported", inferFromVolumeName)
+		return "", "", NewIgnoreableInferenceError(fmt.Errorf("unable to infer defaults from volume %s as type is not supported", inferFromVolumeName))
 	}
 	return "", "", fmt.Errorf("unable to find volume %s to infer defaults", inferFromVolumeName)
 }
 
 func inferDefaultsFromLabels(labels map[string]string, defaultNameLabel, defaultKindLabel string) (defaultName, defaultKind string, err error) {
 	defaultName, hasLabel := labels[defaultNameLabel]
 	if !hasLabel {
-		return "", "", fmt.Errorf("unable to find required %s label on the volume", defaultNameLabel)
+		return "", "", NewIgnoreableInferenceError(fmt.Errorf("unable to find required %s label on the volume", defaultNameLabel))
```

Comment:
We've discussed this offline but I'm still on the fence about ignoring missing labels at all. They might appear later causing future VirtualMachines to look completely different given the same initial definition submitted by the user.

Reply:
I'd argue that an instancetype/preference can change at any point too. You can create two VMs in a row, which can also look completely different.